### ISSUE TITLE
Error with node 14.16.1

### ISSUE THREAD

**Issue by a-ronjohnson** [bytenode/bytenode#136]
Have been successfully using Bytenode for a project on node 12.  Switched to node 14.16.1, and getting the following error when trying to run module after compile with bytenode 1.2.3.  Found same error in #35, so I also tried v1.1.2.

`terminate called after throwing an instance of 'std::out_of_range'
  what():  vector::_M_range_check: __n (which is 0) >= this->size() (which is 0)
Aborted`

Thanks!

**Comment by OsamaAbbas** [comment]
Can you test the same code with Node v13, v15 and v16? Or can you send me (either here or by email) the code that was OK with Node v12 and stopped working with v14?

I have tested a small size project (with express) with v12 to v16 (and with bytenode v1.2.3), and could not reproduce your error.

**Comment by OsamaAbbas** [comment]
I re-tested with a project that have ~50k lines of code in a single file, and it works in all Node versions from 12 to 16.

**Comment by a-ronjohnson** [comment]
v12.22.1 - works
v13.14.0 - works
v14.16.1 - gives error I reported
v15.14.0 - gives error I reported
v16.1.0 - Can't easily test due to incompatible dependency

Used bytenode v1.2.3 for all these

**Comment by a-ronjohnson** [comment]
The module we run calls other modules, so we will see if we can isolate the problem module and send that code along.

**Comment by OsamaAbbas** [comment]
Just to make sure: you know that the `.jsc` files must be re-compiled if you change Node version?

**Comment by a-ronjohnson** [comment]
Yup. We are not re-compiling, but yeah, we compile our source .js files using the same node version that is then used to run the compiles .jsc files.

**Comment by kylebernhardy** [comment]
I have been assisting with trying to isolate what in our code is causing this issue, but I haven't been able to find a consistent lead.  Is there a to debug/analyze the compiled file?

**Comment by OsamaAbbas** [comment]
We have discussed sourcemaps before, here: https://github.com/bytenode/bytenode/issues/60#issuecomment-708561463 .

If this can help, I might re-visit its implementation (because I almost forgot everything in this regard, but the basic idea is discussed there).

Otherwise, I can look into you specific issue by email if you don't mind, which seems an easier and quicker as a first step.
